tests/warc.warc.meta.gz

```diff
@@ -19,16 +19,16 @@ WARC/1.0
 WARC-Date: 2021-11-11T21:11:11Z
 WARC-Concurrent-ID: <urn:uuid:78b1731b-2872-4ab2-92d0-5989d0b40b3a>
 WARC-Type: metadata
-WARC-Record-ID: <urn:uuid:72f91cb9-e32a-4f97-ae09-fe4d89afc2ea>
+WARC-Record-ID: <urn:uuid:b71875c7-d2c8-40ad-ab02-b6ed556246fd>
 WARC-Target-URI: https://www.unt.edu
-WARC-Payload-Digest: sha1:JUGR3BXXMXKXWWJ5PDPGBQT2UIN4O4FY
-WARC-Block-Digest: sha1:JUGR3BXXMXKXWWJ5PDPGBQT2UIN4O4FY
+WARC-Payload-Digest: sha1:NDR4YVM36HMQRAEMWTZB3IFQAYANTOVF
+WARC-Block-Digest: sha1:NDR4YVM36HMQRAEMWTZB3IFQAYANTOVF
 Content-Type: application/warc-fields
-Content-Length: 350
+Content-Length: 352
 
-Identified-Payload-Type: {'fido': 'text/html', 'python-magic': 'text/html'}
-Preservation-Identifier: fmt/471
-Charset-Detected: {'encoding': 'utf-8', 'confidence': 0.99}
-Languages-cld2: {'reliable': True, 'text-bytes': 7896, 'languages': [{'name': 'ENGLISH', 'code': 'en', 'text-covered': 99, 'score': 1070.0}]}
+Identified-Payload-Type: {"fido": "text/html", "python-magic": "text/html"}
+Preservation-Identifier: "fmt/471"
+Charset-Detected: {"encoding": "utf-8", "confidence": 0.99}
+Languages-cld2: {"reliable": true, "text-bytes": 7896, "languages": [{"name": "ENGLISH", "code": "en", "text-covered": 99, "score": 1070.0}]}
```

Comment:
This test file will get updated again when changing warc_metadata_sidcear.py to remove `json.dumps` for `puid` and `soft404`.